rs/crypto/internal/crypto_service_provider/src/vault/local_csp_vault/multi_sig/mod.rs

```diff
@@ -120,8 +119,21 @@ impl<R: Rng + CryptoRng, S: SecretKeyStore, C: SecretKeyStore, P: PublicKeyStore
         &self,
         algorithm_id: AlgorithmId,
         message: &[u8],
-        key_id: KeyId,
     ) -> Result<CspSignature, CspMultiSignatureError> {
+        let commitee_signing_pubkey = self
+            .public_key_store_read_lock()
+            .committee_signing_pubkey()
+            .ok_or(CspMultiSignatureError::PublicKeyNotFound)?;
+        let key_id = KeyId::try_from((
+            AlgorithmId::MultiBls12_381,
+            &commitee_signing_pubkey.key_value,
```

Comment:
Could we define a `impl TryFrom<PublicKeyProto> for KeyId` so it's not necessary to specify the algorithm which is anyway already defined in the proto?

Not a blocker to this PR just seems like it would be a bit nicer.